react-app/cypress.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from 'cypress';
+import vitePreprocessor from 'cypress-vite';
+
+export default defineConfig({
+  e2e: {
+    supportFile: false,
+    baseUrl: 'http://localhost:3000',
```

Comment:
I wish we could change the port using an env var (e.g. `ROMCAL_APP_PORT` or whatever env var name you prefer).

Reply:
asbefore, will do